cmake/kokkoskernels_tpls.cmake

```diff
@@ -109,9 +119,9 @@ MACRO(kokkoskernels_find_header VAR_NAME HEADER TPL_NAME)
     SET(HAVE_CUSTOM_PATHS TRUE)
   ENDIF()
 
-  IF(NOT ${HEADER}_FOUND AND DEFINED KOKKOSKERNELS_${TPL_NAME}_DIR)
+  IF(NOT ${HEADER}_FOUND AND KOKKOSKERNELS_${TPL_NAME}_ROOT)
```

Comment:
What did _DIR need to be changed to _ROOT?

Reply:
We could have left it? It seemed to better match "CMake style" to use _ROOT for the install prefix

Reply:
Was just curious, thanks!